src/main/java/me/wolfyscript/customcrafting/commands/CommandRecipe.java

```diff
@@ -55,20 +49,14 @@ public CommandRecipe(CustomCrafting customCrafting) {
 
     @Override
     public boolean execute(@NotNull CommandSender sender, @NotNull String s, @NotNull String[] args) {
-        if (args.length == 0 && sender instanceof Player) {
-            WolfyUtilities api = customCrafting.getApi();
-            Player p = (Player) sender;
-            InventoryAPI<CCCache> invAPI = api.getInventoryAPI(CCCache.class);
-            if (ChatUtils.checkPerm(p, "customcrafting.cmd.recipes")) {
-                var categories = customCrafting.getDataHandler().getCategories();
-                if (categories.getSortedCategories().size() > 1) {
-                    invAPI.openCluster(p, ClusterRecipeBook.KEY);
-                } else if (!categories.getSortedCategories().isEmpty()) {
-                    invAPI.getGuiHandler(p).getCustomCache().getKnowledgeBook().setCategory(categories.getCategory(0));
-                    invAPI.openGui(p, ClusterRecipeBook.RECIPE_BOOK);
-                }
-            }
+        if (
```

Comment:
I think it would be best to not split the line if it isn't necessary.
Usually, I only split them if the line is too long, and this one still seems fine.

Reply:
Got it. I usually always split for readability but I'll condense this and keep it in mind for future